package-lock.json

```diff
@@ -1324,9 +1324,9 @@
       }
     },
     "node_modules/@npmcli/fs/node_modules/semver": {
-      "version": "7.3.7",
-      "resolved": "https://registry.npmjs.org/semver/-/semver-7.3.7.tgz",
-      "integrity": "sha512-QlYTucUYOews+WeEujDoEGziz4K6c47V/Bd+LjSSYcA94p+DmINdf7ncaUinThfvZyu13lN9OY1XDxt8C0Tw0g==",
+      "version": "7.3.8",
```

Comment:
Is the diff on package-lock needed? Seems like a leftover from running npm update or installing additional packages that are no longer part of this PR...

Reply:
Reverted.